portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -49,30 +56,63 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         if(comments.size() < numComments){
           long id = entity.getKey().getId();
           String text = (String) entity.getProperty("text");
-          long timestamp = (long) entity.getProperty("timestamp");
 
-          Comment comment = new Comment(id,text,timestamp);
+          // Do the translation.
+          Translate translate = TranslateOptions.getDefaultInstance().getService();
```

Comment:
There's no reason to get a new reference to the translator every loop iteration; just create the variable outside the loop.